lib/editor-linter.coffee

```diff
@@ -35,6 +33,10 @@ class EditorLinter
   onDidDestroy: (callback) ->
     return @emitter.on('did-destroy', callback)
 
+  destroy: ->
+    @emitter.emit('did-destroy')
```

Comment:
@steelbrain, he means that it should be done after @deactivate() is called so it represents that deactivate succeeded correctly.